# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/shine/hotfix-arg.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- shine-3.1.1/src/lib/l3mdct.h.vanilla	2026-10-02 19:30:50.669576771 +0300
+++ shine-3.1.1/src/lib/l3mdct.h	2026-10-02 19:31:30.938261688 +0300
@@ -1,7 +1,7 @@
 #ifndef shine_MDCT_H
 #define shine_MDCT_H
 
-void shine_mdct_initialise();
+void shine_mdct_initialise(shine_global_config *config);
 void shine_mdct_sub(shine_global_config *config, int stride);
 
 #endif
